### ISSUE TITLE
classification loss sticks around 2.30212

### ISSUE THREAD

**Issue by qq184861643** [yhenon/pytorch-retinanet#41]
After 1 or 2 epochs of classification loss rightly decreasing, it increases to around 2.30212 and sticks with it and never changes. I'm stuck in this situation now= =
I use my own dataset which re-organized to CSV format as descripted in README.M.
Anybody knows what is going on?

**Comment by zbpjlc** [comment]
i have the same problem,2.3021, classification = torch.clamp(classification, 1e-4, 1.0 - 1e-4)

**Comment by zbpjlc** [comment]
i have the same problem 2.302

**Comment by sherryliang** [comment]
> @zbpjlc low down the lr would solve it, at least works for me

hello，I have the same problem.I would like to know how low your lr is in(optimizer = optim.Adam(shufflenet.parameters(), lr=1e-5))

**Comment by kulkarniniraj** [comment]
I had same problem while transfer learning on my custom dataset, so tried initialization done in model init part. At least both losses are decreasing now.

This is the part I added while init'ing my custom head
```
prior = 0.01
       
self.classificationModel.output.weight.data.fill_(0)
self.classificationModel.output.bias.data.fill_(-math.log((1.0-prior)/prior))

self.regressionModel.output.weight.data.fill_(0)
self.regressionModel.output.bias.data.fill_(0)

self.freeze_bn()
```
